app/Models/EntryDAO.php

```diff
@@ -726,22 +726,24 @@ private function sqlListWhere($type = 'a', $id = '', $state = FreshRSS_Entry::ST
 		$values = array();
 		switch ($type) {
 		case 'a':
-			$where .= 'f.priority > 0 ';
-			$joinFeed = true;
+			$where .= 'f.priority > ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
 			break;
 		case 's':	//Deprecated: use $state instead
+			$where .= 'f.priority >= ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
 			$where .= 'e.is_favorite=1 ';
 			break;
 		case 'c':
+			$where .= 'f.priority >= ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
 			$where .= 'f.category=? ';
 			$values[] = intval($id);
-			$joinFeed = true;
 			break;
 		case 'f':
+			$where .= 'f.priority >= ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
 			$where .= 'e.id_feed=? ';
 			$values[] = intval($id);
 			break;
 		case 'A':
+			$where .= 'f.priority >= ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
 			$where .= '1=1 ';
```

Comment:
The `1=1 ` can now be removed. It was only used to simplify the logic and allow an ` AND` in all cases

Reply:
done

Reply:
Oh, I cannot see the line removed in the git diff. Have you committed already?

Reply:
a minute ago :D